addons/mail/static/src/components/rtc_controller/rtc_controller.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="mail.RtcController" owl="1">
         <div class="o_RtcController">
-            <t t-if="rtcController">
+            <t t-if="rtcController and messaging">
```

Comment:
This one doesn't hurt but make no sense, `rtcController` getter itself already has guard on `messaging`.

Reply:
Indeed, I saw it after it's merged. It's no big deal, the most important is having guard for technical reason. Everything is on their own LOC so when it's time to remove them, it's basically the same amount of LOCs to remove.